www/src/components/repos/SearchRepositories.js

```diff
@@ -49,6 +41,7 @@ export default function SearchRepositories() {
           type="search"
           value={value}
           name="search"
+          icon={<MagnifyingGlassIcon color="text-weak" />}
```

Comment:
One thing I was going to mention to the designers, but the backslash search icon is a bit nicer for us than the plain glass icon, and I've seen a lot of dev-y products start using it.  I think we should keep it.